src/management-system-v2/components/folder-creation-button.tsx

```diff
@@ -34,13 +31,36 @@ const FolderCreationButton: FC<FolderCreationButtonProps> = ({ wrapperElement, .
 
         router.refresh();
         message.open({ type: 'success', content: 'Folder Created' });
-        setModalOpen(false);
+        props.close();
       } catch (e) {
         message.open({ type: 'error', content: 'Something went wrong' });
       }
     });
   };
 
+  return (
+    <FolderModal
+      {...props}
+      open={props.open}
+      close={props.close}
+      spaceId={spaceId}
+      parentId={folderId}
+      onSubmit={createFolder}
+      modalProps={{
+        title: 'Create Folder',
+        okButtonProps: { loading: isLoading },
+      }}
+    />
+  );
+};
+
+const FolderCreationButton: FC<
```

Comment:
I did not find a place where this was used. Can this be removed?

Reply:
I guess it can be removed, I just wanted to keep the button available since it is also available for processes. Should I remove it?

Reply:
I think it is better not to have code in the codebase that is not used at all so i would remove it.